Add unit tests for dog component

stirlaf opened the pull request:
Add unit tests for dog component
Also removed nav and add test debug launch.json

stirlaf committed:
Add unit tests for dog component

Add unit tests for dog component
Also removed nav and add test debug launch.json

pviotti commented:
```
 FAIL  src/Dog.test.tsx
  ● Test suite failed to run

    Cannot find module 'react-testing-library/cleanup-after-each' from 'Dog.test.tsx'

    > 1 | import 'react-testing-library/cleanup-after-each';
        | ^
      2 | import 'jest-dom/extend-expect';
      3 | 
      4 | import { Dog } from './Dog';

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:221:17)
      at Object.<anonymous> (src/Dog.test.tsx:1:1)
```
I guess it's missing react-testing-library as a dependency.  
(the CI/CD build succeeds because tests are not run, at the moment..)

stirlaf committed:
Add missing dependencies.

Add missing dependencies.

stirlaf commented:
> ```
>  FAIL  src/Dog.test.tsx
>   ● Test suite failed to run
> 
>     Cannot find module 'react-testing-library/cleanup-after-each' from 'Dog.test.tsx'
> 
>     > 1 | import 'react-testing-library/cleanup-after-each';
>         | ^
>       2 | import 'jest-dom/extend-expect';
>       3 | 
>       4 | import { Dog } from './Dog';
> 
>       at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:221:17)
>       at Object.<anonymous> (src/Dog.test.tsx:1:1)
> ```
> I guess it's missing react-testing-library as a dependency.
> (the CI/CD build succeeds because tests are not run, at the moment..)

Oops, thanks for the reminder .  I reverted my dependency changes temporarily and forgot to add the dependencies back before creating the pull request.  I've added them back again now.
